### PR TITLE
11.0 mig account due list payment mode

### PR DESCRIPTION
**Pull request by cubells**
Depends on 
- [x] account_due_list #170 

- Update README.rst
- Update translation

cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
@cubells, please cleanup the commit history this way:

seleccion_006

File: account_due_list_payment_mode/README.rst [hunk 1]
```diff
@@ -16,6 +18,8 @@ This module depends on:
 * account_due_list
 * account_payment_partner
 
+In order to see payments, you must install account_invoicing app.
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/README.rst]: sale_management?

Comment (pedrobaeza) [reply account_due_list_payment_mode/README.rst]: Put it under "Configuration" section.

Comment (pedrobaeza) [reply account_due_list_payment_mode/README.rst]: And you need to check "Show full accounting features".

File: account_due_list_payment_mode/README.rst [hunk 2]
```diff
@@ -24,18 +28,21 @@ There is nothing to configure.
...
 
 Bug Tracker
 ===========
 
-Bugs are tracked on `GitHub Issues <https://github.com/OCA/account-payment/issues>`_.
-In case of trouble, please check there if your issue has already been reported.
-If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
-`here <https://github.com/OCA/account-payment/issues/new?body=module:%20account_due_list_payment_mode%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/account-payment/issues>`_. n case of trouble, please
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/README.rst]: s/n case/In case

File: account_due_list_payment_mode/README.rst [hunk 3]
```diff
@@ -47,16 +54,19 @@ Contributors
 * cubells <vicent@vcubells.net>
 * Sergio Teruel <sergio.teruel@tecnativa.com>
 * Albert De La Fuente <info@haevas.com>
+* Vicent Cubells <vicent.cubells@tecnativa.com>
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/README.rst]: You are already above

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
Please rebase and fix what stated before now that dependency is merged.

**Review by pedrobaeza — APPROVED** [review]

**Review by luismontalba — APPROVED** [review]
Tested on runbot

**Pull request merged**